.style.yapf

```diff
@@ -1,5 +1,5 @@
 [style]
-# For docs, see https://github.com/google/yapf/blob/master/README.rst
+# For docs, see https://github.com/google/yapf/blob/main/README.rst
```

Comment:
Confimed, this is the correct URL on the yapf repo too.

Other links to external projects that use `master` are outside of our control, of course.